doc/UsersGuide/BuildingRunningTesting/Tutorial.rst

```diff
@@ -574,13 +574,277 @@ A line of severe storms brought strong winds, flash flooding, and tornadoes to t
+
+Experiment 1: Control
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Edit the configuration file (``config.yaml``) to include the variables and values in the sample configuration excerpts below. 
+
+.. Hint:: 
+   
+   To open the configuration file in the command line, users may run the command:
+
+   .. code-block:: console
+
+      vi config.yaml
+
+   To modify the file, hit the ``i`` key and then make any changes required. To close and save, hit the ``esc`` key and type ``:wq`` to write the changes to the file and exit/quit the file. Users may opt to use their preferred code editor instead.
+
+Start in the ``user:`` section and change the ``MACHINE`` and ``ACCOUNT`` variables. For example, when running on a personal MacOS device, users might set:
+
+.. code-block:: console
+
+   user:
+      RUN_ENVIR: community
+      MACHINE: macos
+      ACCOUNT: none
+
+For a detailed description of these variables, see :numref:`Section %s <user>`.
+
+Users do not need to change the ``platform:`` section of the configuration file for this tutorial. The default parameters in the ``platform:`` section pertain to METplus verification, which is not addressed here. For more information on verification, see :numref:`Section %s <VXCases>`.
+
+In the ``workflow:`` section of ``config.yaml``, update ``EXPT_SUBDIR`` and ``PREDEF_GRID_NAME``.
+
+.. code-block:: console
+
+   workflow:
+     USE_CRON_TO_RELAUNCH: false
+     EXPT_SUBDIR: control
+     CCPP_PHYS_SUITE: FV3_GFS_v16
+     PREDEF_GRID_NAME: RRFS_CONUS_13km
+     DATE_FIRST_CYCL: '2019102812'
+     DATE_LAST_CYCL: '2019102812'
+     FCST_LEN_HRS: 12
+     PREEXISTING_DIR_METHOD: rename
+     VERBOSE: true
+     COMPILER: intel
+
+.. _CronNote:
+
+.. note::
+
+   Users may also want to set ``USE_CRON_TO_RELAUNCH: true`` and add ``CRON_RELAUNCH_INTVL_MNTS: 3``. This will automate submission of workflow tasks when running the experiment. However, not all systems have :term:`cron`. 
+
+``EXPT_SUBDIR:`` This variable can be changed to any name the user wants from "gfsv16_physics_fcst" to "forecast1" to "a;skdfj". However, the best names will indicate useful information about the experiment. This tutorial uses ``control`` to establish a baseline, or "control", forecast. Since this tutorial helps users to compare the output from two different forecasts --- one that uses the FV3_GFS_v16 physics suite and one that uses the FV3_RRFS_v1beta physics suite --- "gfsv16_physics_fcst" could be a good alternative directory name.
+
+``PREDEF_GRID_NAME:`` This experiment uses the RRFS_CONUS_13km, rather than the default RRFS_CONUS_25km grid. Using the RRFS_CONUS_13km grid allows for fewer computational restraints compared to the 25km grid.  For more information on this grid, see :numref:`Section %s <RRFS_CONUS_13km>`.
```

Comment:
```suggestion
``PREDEF_GRID_NAME:`` This experiment uses the RRFS_CONUS_13km, rather than the default RRFS_CONUS_25km grid. Using the RRFS_CONUS_13km grid provides a higher resolution forecast, more detailed forecast; however, it is more computationally expensive compared to the 25km grid.  For more information on this grid, see :numref:`Section %s <RRFS_CONUS_13km>`.
```
@jdkublnick Technically, there are more computational restraints with `RRFS_CONUS_13km` because it requires more compute power to run at a higher resolution (note that higher resolution translates to smaller grid cells and more of them. 13km is the approximate grid cell size, and there will be more of those on the CONUS compared to the 25km grid cells on the lower resolution `RRFS_CONUS_25km` domain). 
@MichaelLueken Does my change above make sense, or can you refine it?

Reply:
@gspetro-NOAA - Your modification makes sense.  The use of the 13km grid versus the 25km grid is for a higher resolution, more detailed forecast.  The 13 km grid will also require more resources in order to run, for the reason you noted above.  The only modification I would make would be to remove forecast after higher resolution:
```suggestion
``PREDEF_GRID_NAME:`` This experiment uses the RRFS_CONUS_13km, rather than the default RRFS_CONUS_25km grid. Using the RRFS_CONUS_13km grid provides a higher resolution, more detailed forecast; however, it is more computationally expensive compared to the 25km grid.  For more information on this grid, see :numref:`Section %s <RRFS_CONUS_13km>`.
```